lib/motor.js

```diff
@@ -646,7 +646,7 @@ Motor.prototype.setPin = function(pin, value) {
 };
 
 Motor.prototype.setPWM = function(pin, value) {
-  this.io.analogWrite(pin, value);
+  this.io.analogWrite(pin, __.map(value, 0, 255, 0, this.board.RESOLUTION.PWM));
```

Comment:
Can we do a pass on this file and change `__` to `Fn`? Probably in another patch?